spec/presenters/link_api_response_presenter_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe LinkApiResponsePresenter do
+  describe '#present' do
+    let(:authority) { FactoryGirl.build(:local_authority) }
+    let(:presenter) { described_class.new(authority, link) }
```

Comment:
If I remember @h-lame correctly the `presenter` is called on line 27 and at that point `link` is available from line 9. Using outside this context it would indeed break.